tt_metal/core_descriptors/grayskull_120_arch.yaml

```diff
@@ -17,8 +17,27 @@ E150:
     storage_cores: # Relative to grid of tensix cores
       [[1, -1],[2, -1],[3, -1],[4, -1],[5, -1],[7, -1],[8, -1],[9, -1],[10, -1],[11, -1]]
 
-    dispatch_cores: # Relative to grid of tensix cores
-      [[0, -1],[6, -1]]
+    producer_cores:
+      [[0, -1]]
+
+    consumer_cores:
+      [[6, -1]]
+  2:
+    l1_bank_size:
+      524288
+
+    compute_with_storage_grid_range: # Logical only start and end [x, y]
+      start: [0, 0]
+      end: [11, 8]
+
+    storage_cores: # Relative to grid of tensix cores
+      [[2, -1],[3, -1],[4, -1],[5, -1],[8, -1],[9, -1],[10, -1],[11, -1]]
+
+    producer_cores:
+      [[0, -1], [1, -1]]
+
+    consumer_cores:
```

Comment:
also, don't we need to support E75 for project grayskull? I see it still has "dispatch_cores"

Reply:
also need to update WH yamls

Reply:
Fixed.